app/views/govuk_publishing_components/components/docs/accordion.yml

```diff
@@ -183,7 +183,7 @@ examples:
               </ul>'
   with_data_attributes:
     description: |
-      Users can add multiple custom attributes to the accordion to each section. Accepts an hash, so multiple attributes can be added.
+      Adds custom data attributes to each section of the accordion. Accepts an hash, so multiple attributes can be added.
```

Comment:
I think this should be `Accepts a hash`?

Reply:
This is the current sentence

`Adds custom data attributes to each section of the accordion. *Accepts an hash*, so multiple attributes can be added.`

Do you mean that this sentence above should perhaps drop or move the first part?

Eg just: `Accepts an hash, so multiple attributes can be added.`

Reply:
No I think it should be `Accepts a hash` and not `Accepts an hash`.

Reply:
Ahh! Ace, updated